src/GestureAnimationDirector.vala

```diff
@@ -96,8 +96,15 @@ public class Gala.GestureAnimationDirector : Object {
         handlers.clear ();
     }
 
-    public static float animation_value (float initial_value, float target_value, int percentage) {
-        return (((target_value - initial_value) * percentage) / 100) + initial_value;
+    public static float animation_value (float initial_value, float target_value, int percentage, bool rounded = false) {
```

Comment:
Are there animations you don't want to round off?

Reply:
Animations that involve scaling, like the multitasking view desktop animation, use a value between 0.0 and 1.0. Rounding that value breaks the animation.
Also, when scaling a window down, the window may not have a size divisible by physical pixels. That's why the `rounded` flag is not set in those cases.

Reply:
Awesome, thank you for explaining. Would you mind though just writing that as a comment on the function? :) That way a year from now when we or someone else looks at the code, we will know the reason behind this

Reply:
I just documented the method. Thanks for the review Felipe.